### ISSUE TITLE
The use of content labels 

### ISSUE THREAD

**Issue by Krakert** [Krakert/Crypto-Tracker#27]
Missing Readable Labels for Screen Readers
![image](https://github.com/Krakert/Crypto-Tracker/assets/61704774/df9e89c1-3648-4a5c-9240-5313a3e61800)

 The specific element with the path `android:id/content/r0[1]/View[1]/Button[1]` appears to potentially lack a readable label for screen readers.

Device Specifications:

Model Name: Google Pixel Watch
Manufacturer: Google
Android Version: Android 11 (SDK 30)
Locale: en_US
Screen Size: 384 x 384
Screen Density (DPI): 320
RAM (Total Memory): 2,048 MB
OpenGL ES Version: 3.1
ABI: armeabi-v7a
CPU: -
